### PR TITLE
upload node-dpr-aware-loading project

### PR DESCRIPTION
**Pull request by anton-karlovskiy**

### CODE REVIEW

**Comment by addyosmani** [comment]
Similar comment to #43 regarding consistency in the body background color. Can we adjust background to `#282c34`?

image

**Comment by addyosmani** [comment]
DPR switching images appears to be working as expected 👍 

image

File: node-dpr-aware-loading/functions/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+const functions = require('firebase-functions');
+
+// // Create and Deploy Your First Cloud Functions
+// // https://firebase.google.com/docs/functions/write-firebase-functions
+//
+// exports.helloWorld = functions.https.onRequest((request, response) => {
+//  response.send("Hello from Firebase!");
+// });
+
```
Comment (addyosmani) [inline node-dpr-aware-loading/functions/index.js]: Let's remove the boilerplate comments for cloud functions here?

File: node-dpr-aware-loading/functions/index.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+app.disable('x-powered-by');
+app.use(express.static(path.join(__dirname, 'build')));
+app.set('views', __dirname + '/build');
+app.engine('html', require('ejs').renderFile);
+app.set('view engine', 'html');
+
+// check requests
+// const morgan = require('morgan');
+// app.use(morgan('combined'));
+
```
Comment (addyosmani) [inline node-dpr-aware-loading/functions/index.js]: Let's remove the comments

File: node-dpr-aware-loading/server.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+app.disable('x-powered-by');
+app.use(express.static(path.join(__dirname, 'public')));
+app.set('views', __dirname + '/public');
+app.engine('html', require('ejs').renderFile);
+app.set('view engine', 'html');
+
+// check requests
+// const morgan = require('morgan');
+// app.use(morgan('combined'));
+
```
Comment (addyosmani) [inline node-dpr-aware-loading/server.js]: Remove the comments

File: node-dpr-aware-loading/functions/index.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+// // Create and Deploy Your First Cloud Functions
+// // https://firebase.google.com/docs/functions/write-firebase-functions
+//
+// exports.helloWorld = functions.https.onRequest((request, response) => {
+//  response.send("Hello from Firebase!");
+// });
+
+const express = require('express');
+// const bodyParser = require('body-parser');
```
Comment (addyosmani) [inline node-dpr-aware-loading/functions/index.js]: Remove (commented out)

File: node-dpr-aware-loading/functions/index.js (hunk 1 above)
Comment (anton-karlovskiy) [reply node-dpr-aware-loading/functions/index.js]: Yes, for clean codes.

File: node-dpr-aware-loading/functions/index.js (hunk 2 above)
Comment (anton-karlovskiy) [reply node-dpr-aware-loading/functions/index.js]: Yes for clean codes.

File: node-dpr-aware-loading/server.js (hunk 3 above)
Comment (anton-karlovskiy) [reply node-dpr-aware-loading/server.js]: Yes for clean codes.

File: node-dpr-aware-loading/functions/index.js (hunk 4 above)
Comment (anton-karlovskiy) [reply node-dpr-aware-loading/functions/index.js]: Yes for clean codes.

**Comment by anton-karlovskiy** [comment]
All comments are fixed!

**Pull request merged**